lib/screens/common/sidenavbar.dart

```diff
@@ -22,8 +23,16 @@ class Sidenav extends StatelessWidget {
   final double height;
   final double width;
   final String username;
+  final String mobile;
+  final String email;
   final String referralCode;
-  Sidenav({this.height, this.width, this.username, this.referralCode});
+  Sidenav(
+      {this.height,
+      this.width,
+      this.username,
+      this.referralCode,
+      this.mobile,
+      this.email});
```

Comment:
should not all of these parameters be `@required`, we need all of them to make the Sidenav work ?

Reply:
I didnt required bcz i think in future only sidenav will be required at some place will add.

Reply:
on reviewing again, i found that this function is getting used on referral code screen and we cant send these things from there i.e. y i feel required is not rqd.